scripts/hook-governance/evaluate_docstring_ratchet.py

```
@@ -0,0 +1,252 @@
+
+
+def load_structured_file(path: Path) -> object:
+    raw_text = path.read_text(encoding="utf-8")
+    try:
+        return json.loads(raw_text)
+    except json.JSONDecodeError:
+        if yaml is not None:
+            return yaml.safe_load(raw_text)
+    raise DocstringRatchetError(
+        f"unable to parse structured file '{path}' as JSON or YAML"
+    )
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)